Tasks/VsTestV2/runvstest.ts

```diff
@@ -7,18 +7,20 @@ import * as ci from './cieventlogger';
 import * as utils from './helpers';
 import * as inputParser from './inputparser';
 import * as os from 'os';
+import * as localtest from './vstest';
+
+const request = require('request');
 const osPlat: string = os.platform();
 
 tl.setResourcePath(path.join(__dirname, 'task.json'));
 
-if (osPlat !== 'win32') {
-    // Fail the task if os is not windows
-    tl.setResult(tl.TaskResult.Failed, tl.loc('OnlyWindowsOsSupported'));
-} else {
-    //Starting the VsTest execution
+async function execute() {
     const taskProps = { state: 'started', result: '' };
     ci.publishEvent(taskProps);
 
+    const enableApiExecution = await getFeatureFlag(tl.getVariable('System.TeamFoundationCollectionUri'),
```

Comment:
>getFeatureFlag [](start = 37, length = 14)

rename it to isFeatureFlagEnabled.

getFF should ideally return on and off

Reply:
done